charts/team-ns/templates/argocd/argocd-applicationset.yaml

```diff
@@ -18,8 +18,9 @@ spec:
   template:
     metadata:
       name: 'team-{{ $v.teamId }}-{{ .name }}'
-      {{- if or (eq $a.type "digest") (eq $a.type "semver") }}
       annotations:
+        argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
```

Comment:
Looking at description of https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#skip-dry-run-for-new-custom-resources-types I am wondering why it is necessary. Have you encountered charts with operators that they deploy CRDs?

Reply:
seems that Nvidia operator deploys CRDs